recipe/build.sh

```diff
@@ -164,12 +191,19 @@ elif [[ ${cuda_compiler_version} != "None" ]]; then
             echo "unknown CUDA arch, edit build.sh"
             exit 1
     esac
+    # Warning from pytorch v1.12.1: In the future we will require one to
+    # explicitly pass TORCH_CUDA_ARCH_LIST to cmake instead of implicitly
+    # setting it as an env variable.
```

Comment:
Isn't that irrelevant to calling `setup.py`, though? After all, `setup.py` is passing it explicitly to CMake.

Reply:
I seem to remember seeing the error, also via setup.py, will investigate further

Reply:
I don't think this variable gets passed via setup.py, it gets detected in a cmake module https://github.com/pytorch/pytorch/blob/266fd35c5842902f6304aa8e7713b252cbfb243c/cmake/public/utils.cmake#L327

Setting it via CMAKE_ARGS should solve this, have changed it to that